README.md

```diff
@@ -59,10 +61,20 @@ The following flavors of ecAudit are no longer maintained.
 
 Install and configure ecAudit using the setup guide for your Cassandra version.
 
-* [ecAudit Setup Guide for Cassandra 3.11.\<latest>](https://github.com/Ericsson/ecaudit/blob/master/doc/setup.md)
+* [ecAudit Setup Guide for Cassandra 4.1.\<latest>](https://github.com/Ericsson/ecaudit/blob/master/doc/setup.md)
+* [ecAudit Setup Guide for Cassandra 4.0.\<latest>](https://github.com/Ericsson/ecaudit/blob/release/c4.0/doc/setup.md)
+* [ecAudit Setup Guide for Cassandra 3.11.\<latest>](https://github.com/Ericsson/ecaudit/blob/release/c3.11/doc/setup.md)
 * [ecAudit Setup Guide for Cassandra 3.0.\<latest>](https://github.com/Ericsson/ecaudit/blob/release/c3.0/doc/setup.md)
-* [ecAudit Setup Guide for Cassandra 3.0.11](https://github.com/Ericsson/ecaudit/blob/release/c3.0.11/doc/setup.md)
-* [ecAudit Setup Guide for Cassandra 2.2.\<latest>](https://github.com/Ericsson/ecaudit/blob/release/c2.2/doc/setup.md)
+
+
+### Upgrade
+
+Upgrade ecAudit using then upgrade guide for your Cassandra version
+
+* [ecAudit Upgrade Guide for Cassandra 4.1.\<latest>](https://github.com/Ericsson/ecaudit/blob/master/UPGRADING.md)
+* [ecAudit Upgrade Guide for Cassandra 4.0.\<latest>](https://github.com/Ericsson/ecaudit/blob/release/c4.0/UPGRADING.md)
+* [ecAudit Upgrade Guide for Cassandra 3.11.\<latest>](https://github.com/Ericsson/ecaudit/blob/release/c3.11/UPGRADING.md)
+* [ecAudit Upgrade Guide for Cassandra 3.0.\<latest>](https://github.com/Ericsson/ecaudit/blob/release/c3.0/doc/UPGRADING.md)
```

Comment:
Wrong link for upgrade doc for 3.0

Should be https://github.com/Ericsson/ecaudit/blob/release/c3.0/UPGRADING.md

Reply:
Done.